WordPress/src/main/java/org/wordpress/android/ui/comments/CommentActionResult.java

```diff
@@ -0,0 +1,19 @@
+package org.wordpress.android.ui.comments;
+
+public class CommentActionResult {
+
+    public static int COMMENT_ID_ON_ERRORS = -1;
+    public static int COMMENT_ID_UNKNOWN = -2;
```

Comment:
Am I correct in thinking that `COMMENT_ID_UNKNOWN` is used primarily for replies, when the commentID isn't known yet? If so, a comment stating this purpose would be helpful (otherwise it might be assumed `COMMENT_ID_UNKNOWN` should be treated as an error, which is what I initially thought).